operators/deploy/crownlabs-delete-labinstance/cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+--- 
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata: 
+  labels: 
+    app: control-old
+  name: delete-crono
+  namespace: delete-space
+spec: 
+  jobTemplate: 
+    spec: 
+      template: 
+        spec: 
+          containers: 
+            - 
+              image: "chiaraoggeri/delete-repo:v3"
```

Comment:
I would add this component to the build pipeline (just add crownlabs-delete-labinstance in this list https://github.com/netgroup-polito/CrownLabs/blob/d6fbcbcd487d6cdec4870ec1f431c3196a09b99b/.github/workflows/build.yml#L92 should do the work) and then link this image to the one the pipeline pushes, instead of a private one.